models/interactions.sql

```diff
@@ -0,0 +1,195 @@
+{{ 
+  config(
+    materialized='table',
+    unique_key = 'event_id',
+    sort = 'start_tstamp',
+    dist = 'event_id'
+  )
+}}
+
+with prep as (
+
+ 	select
+    e.event_id,
+    e.page_view_id,
+    e.domain_sessionid,
+    e.domain_userid,
+    coalesce(y.player_id, me.html_id, r.content_title) as media_id, -- replace content_title with id field when available
+    round(mp.duration) as duration,
+    coalesce(y.player_id, me.html_id, r.content_title) as title, -- replace ids with title fields when available
+    case when me.media_type = 'audio' then 'audio' else 'video' end as media_type,
```

Comment:
Why not just get this info from `ref('plays_by_pageview')`?

Reply:
we can't, interactions get aggregated into plays_by_pageview so it has to be calculated upstream